addon-resizer/nanny/main/pod_nanny.go

```diff
@@ -53,7 +53,7 @@ var (
 
 func main() {
 	// First log our starting config, and then set up.
-	log.Infof("Invoked by %v", os.Args)
+	log.V(1).Infof("Invoked by %v", os.Args)
```

Comment:
This contradicts logging guide, where configuration details should be under V(1).

However, I tend to agree that O(1) logs should be on level 0. ACK